concent_api/utils/api_view.py

```diff
@@ -0,0 +1,59 @@
+from functools import wraps
+from base64    import b64decode
+
+from django.http                  import JsonResponse, HttpResponse
+from django.conf                  import settings
+from django.views.decorators.csrf import csrf_exempt
+from golem_messages.message       import Message
+from golem_messages               import dump, load
+
+
+class Http400(Exception):
+    pass
+
+
+def api_view(view):
+    @wraps(view)
+    @csrf_exempt
+    def wrapper(request, *args, **kwargs):
+        client_public_key = b64decode(request.META['CONCENT_CLIENT_PUBLIC_KEY'].encode('ascii'))
+
+        if request.content_type not in ['application/octet-stream', '', 'application/json']:
+            return JsonResponse({'error': 'Concent supports only application/octet-stream and application/json.'}, status = 415)
+
+        if len(request.body) == 0:
+            message = None
+        else:
+            if request.content_type == '':
+                raise Http400('Content-Type is missing.')
+            if request.content_type == 'application/json':
+                message = request.body
+            if request.content_type == 'application/octet-stream':
+                message = load(
+                    request.body,
+                    settings.CONCENT_PRIVATE_KEY,
+                    client_public_key
+            )
```

Comment:
Please always run `./full-check.py` before submitting a pull request. This like generates a warning:

```
./concent_api/utils/api_view.py:36:13: E122 continuation line missing indentation or outdented
```